salt/modules/kapacitor.py

```diff
@@ -14,16 +14,18 @@
 
 .. versionadded:: 2016.11.0
 '''
-from __future__ import absolute_import
-
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
 import logging
 
+# Import Salt lobs
+from salt.ext import six
 import salt.utils.http
 import salt.utils.json
 import salt.utils.path
 from salt.utils.decorators import memoize
 
-LOG = logging.getLogger(__name__)
+log = logging.getLogger(__name__)
```

Comment:
Since we don't actually _use_ the log object in this file, we can remove lines 27 and 28.